torch/distributions/kl.py

```diff
@@ -665,7 +665,16 @@ def _kl_normal_gumbel(p, q):
     t3 = torch.exp(-mean_scale_ratio + 0.5 * var_scale_sqr_ratio + loc_scale_ratio)
     return -t1 + t2 + t3 - (0.5 * (1 + math.log(2 * math.pi)))
 
-# TODO: Add Normal-Laplace KL Divergence
+
+@register_kl(Normal, Laplace)
+def _kl_normal_laplace(p, q):
+    loc_diff = p.loc - q.loc
+    scale_ratio = p.scale / q.scale
+    loc_diff_scale_ratio = loc_diff / p.scale
+    t1 = 0.5 * torch.log(0.5 * math.pi * scale_ratio.pow(2))
```

Comment:
IMHO, `0.5 * torch.log(0.5 * math.pi) + torch.log(scale_ratio)` would be preferable because we don't need to square it. I think this can lead to better numerical stability when the 'scale ratio' is too small or too large.

Reply:
Good point, thank you! I have made the corresponding changes.